release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1302,7 +1302,7 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, when using the OpenShift SDN cluster network provider, idled services used an increasing amount of CPU to un-idle services. In this release, the idling code for kube-proxy is optimized to reduce CPU utilizing for service idling. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966521[BZ#1966521])
 
-* Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])    
+* Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])
```

Comment:
Rephrase suggestion:

... the presence of any unknown field in an internal configuration map could fail to start the OVN-Kubernetes pods to fail to start during a cluster upgrade.

Reply:
Updated. Thanks.

Reply:
But I agree with your edit. Don't have time to follow up on this.